app/views/users/index.html.erb

```diff
@@ -13,11 +13,9 @@
 
 <% @users.each do |user| %>
   <tr>
-    <td><%= user.name %></td>
+    <td><%= link_to user.name, user_path(user) %></td>
```

Comment:
Fair.  The Terms page is a mostly-useless mess, mostly because Terms are themselves mostly inert.  That page above shows literally all the information contained in a Term object.  Whereas the Users index page does not.  And the Terms#show page is even dumber.

I think for now the right design is to have the Users#index page have three links per row, Show / Edit / Impersonate, and have the Terms#index page just have Edit.